ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html

```diff
@@ -26,6 +26,8 @@
      data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
      class="cmp-adaptiveform-textinput"
      data-cmp-is="adaptiveFormTextInput"
+     data-cmp-visible="${text.visible ? 'true' : 'false'}"
```

Comment:
is it not required in datepicker as well?

Reply:
It must be. I would be adding these in all the components